index.bs

```diff
@@ -1406,7 +1406,7 @@ Instances of <code>WritableStream</code> are created with the internal slots des
 <ol>
   <li> Assert: <var>stream</var>@\[[state]] is <code>"closing"</code>.
   <li> Let <var>sinkClosePromise</var> be PromiseInvokeOrNoop(<var>stream</var>@\[[underlyingSink]],
-    <code>"close"</code>).
+    <code>"close"</code>, «‍»).
```

Comment:
I added this since I saw it in the algorithm of WritableStream.abort(reason). Shall we remove it too and make PromiseInvokeOrFallbackOrNoop's argument omittable but only args2?